wordpress-mobile/gutenberg-mobile
# [Android] Disables saving react root view from parent

*Related PRs:*
**WordPress-Android PR:** https://github.com/wordpress-mobile/WordPress-Android/pull/20046
**Gutenberg PR:** https://github.com/WordPress/gutenberg/pull/58266

To test:
See https://github.com/wordpress-mobile/WordPress-Android/pull/20046

PR submission checklist:
